tests/analysis/test_tracking_quality.py

```diff
@@ -69,8 +69,8 @@ def test_tracking_quality_day(self, testdata_sp_tpls_geolife_long):
         # test if the result of the user agrees
         quality = ti.analysis.tracking_quality.temporal_tracking_quality(sp_tpls, granularity="day")
 
-        assert quality_manual == quality.loc[(quality["user_id"] == 0) & (quality["day"] == 0), "quality"].values[0]
-        assert (quality["quality"] < 1).all()
+        assert quality_manual == quality.iloc[0]["quality"]
+        assert (quality["quality"] <= 1).all()
```

Comment:
Why did we need `<=` instead of `<` that was there before?

Reply:
This relates to a bug we introduced before, check #221. So quality theoretically could take the value 1. The change here is just for the "correctness" and did not influence test results